packages/issuer/package.json

```diff
@@ -11,14 +11,14 @@
   "dependencies": {
     "@sphereon/openid4vci-common": "workspace:../common",
     "@sphereon/ssi-types": "^0.9.0",
-    "jose": "^4.10.0",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@sphereon/openid4vci-client": "workspace:../client",
     "@types/jest": "^29.5.0",
     "@types/uuid": "^9.0.1",
-    "nock": "^13.2.9"
+    "nock": "^13.2.9",
+    "jose": "^4.10.0"
```

Comment:
I would also expect the jose dep to be removed from the common module (or moved to dev deps if it is used in tests)

Reply:
Actually it's from the client module. Fixed